imports/wasi_snapshot_preview1/wasi_stdlib_test.go

```diff
@@ -457,3 +458,38 @@ func testHTTP(t *testing.T, bin []byte) {
 	console := <-ch
 	require.Equal(t, "", console)
 }
+
+func Test_Stdin(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("Nonblocking stdio is not supported on wasip1+windows.")
+	}
+	toolchains := map[string][]byte{}
+	if wasmGotip != nil {
+		toolchains["gotip"] = wasmGotip
+	}
+
+	for toolchain, bin := range toolchains {
+		toolchain := toolchain
+		bin := bin
+		t.Run(toolchain, func(t *testing.T) {
+			testStdin(t, bin)
+		})
+	}
+}
+
+func testStdin(t *testing.T, bin []byte) {
+	r, w, err := os.Pipe()
+	require.NoError(t, err)
+	moduleConfig := wazero.NewModuleConfig().
+		WithSysWalltime().WithSysNanotime(). // HTTP middleware uses both clocks
```

Comment:
This comment doesn't seem relevant here, was it leftover from a copy/paste?
```suggestion
		WithSysWalltime().WithSysNanotime().
```

Reply:
leftover from copypaste 😬